client/src/components/course-results-view/SisuDownloadDialog.tsx

```diff
@@ -197,20 +214,29 @@ export default function SisuDownloadDialog(props: {
               />
             </Box>
             {(exportedValuesInList()) && (
-              <Box sx={{ ml: 1, my: 2 }}>
-                <FormControlLabel control={(
-                  <Checkbox
-                    id="select-all"
-                    size="small"
-                    onClick={(): void => setOverride(!override)}
-                    checked={override} />
-                )} label={(
-                  <Typography variant='body2' sx={{ color: 'red' }}>
-                    The list includes grades already exported to Sisu.
-                    Please check the box if you want to include these previously exported grades.
-                  </Typography>
-                )
-                } />
+              <Box sx={{ ml: 1, my: 2, mr: 10 }}>
+                <Typography variant='body2' sx={{ color: 'red' }}>
+                  The list includes grades already exported to Sisu.
```

Comment:
I don't expect this to be fixed in this PR, but this should actually be student specific and not grade specific. The described use case for this feature was to cover the following scenario:

Say that student X gets grade 4, and this grade is uploaded to Sisu. Student X wants to raise their grade, and does some extra work to raise their grade to a 5. A new final grade is calculated for student X. Eventually the teacher wants to upload more grades to Sisu, including student X who has already been uploaded before, along with other students who do not have a grade in Sisu yet.

With this current implementation, Grades will not inform the teacher that this student should already have a grade in Sisu. So when the teacher goes to upload this CSV, it will probably fail because teachers themselves are not allowed to raise the grades of students in Sisu and must send off that information for someone else to handle.

Reply:
On further testing this dialog does appear to work correctly on a student basis, but the course results table does not.